adal/authentication_context.py

```diff
@@ -95,6 +101,7 @@ def __init__(
             'options': GLOBAL_ADAL_OPTIONS,
             'api_version': api_version,
             'verify_ssl': None if env_value is None else not env_value, # mainly for tracing through proxy

```

Comment:
Also, this line will need to be updated, otherwise the argument value will not be propagated particularly when it is a string to a bundle

Reply:
Actually, the argument value was passed as default parameter in the `os.environ`, so yes, it was propagated